### PR TITLE
Add replace SCWarrants with NoRamp flag.  Bug fix to Calibrator in NXWarrants

### PR DESCRIPTION
**Pull request by petecressman**

### CODE REVIEW

File: releasenotes/jmri4.5.6.shtml [hunk 1]
```diff
@@ -417,6 +417,54 @@ The <a href="https://github.com/JMRI/JMRI/issues?utf8=✓&q=is%3Aclosed&q=milest
 
    <h4>Web Access</h4>
         <ul>
+            <li></li>
```
Comment (mattharris) [inline releasenotes/jmri4.5.6.shtml]: @petecressman it seems that you might have started from an earlier version of the release note that had already had the placeholder entries stripped out.

Could you revert the additions of these placeholder entries please?

Comment (petecressman) [inline releasenotes/jmri4.5.6.shtml]: I would be glad to do that, but I don't know exactly what "revert the additions" means.  I will resubmit with all the empty placeholders removed.  If that is incorrect, please advise.
Thanks, sorry i messed it up.
Pete C

**Comment by bobjacobsen** [comment]
> it seems that you might have started from an earlier version of the release note that had already had the placeholder entries stripped out.
> 
> Could you revert the additions of these placeholder entries please?

I’m curious how this can get past the Git merge procedure.  Normally, starting with something older will either get a correct merge, or will throw a conflict.  Can we understand how we ended up with old content this time?

Bob

**Comment by pabender** [comment]
On Nov 29, 2016, at 8:36 PM, Bob Jacobsen <notifications@github.com> wrote:

> > it seems that you might have started from an earlier version of the release note that had already had the placeholder entries stripped out.
> > 
> > Could you revert the additions of these placeholder entries please?
> 
> I’m curious how this can get past the Git merge procedure. Normally, starting with something older will either get a correct merge, or will throw a conflict. Can we understand how we ended up with old content this time?

For some reason, I think this has to do with the way certain IDEs handle the merge process.  

We ended up having to revert a commit Dan Boudreau made last week ( or was it the week before ) because of similar issues ( though much more widespread, and causing test errors...)

Paul

**Comment by bobjacobsen** [comment]
> For some reason, I think this has to do with the way certain IDEs handle the merge process.  
> 
> We ended up having to revert a commit Dan Boudreau made last week ( or was it the week before ) because of similar issues ( though much more widespread, and causing test errors...)

That makes it even more scary, unfortunately.  This time, we noticed that changes had been (accidentally/unintentionally, but wrongly) reverted.  But there might be cases where we don’t notice, and work gets lost.

**Comment by rhwood** [comment]
I'm thinking that its likely that the IDE is allowing the user to keep their working changes when there is a conflict between an incoming merge and the working copy instead of requiring a manual review.

**Comment by petecressman** [comment]
I did a pull before editing and it appeared to merge, yet there were all place holders.  Subsequent pulls say everything is up to date.Pete
 

    On Tuesday, November 29, 2016 6:00 PM, Randall Wood <notifications@github.com> wrote:
 

 I'm thinking that its likely that the IDE is allowing the user to keep their working changes when there is a conflict between an incoming merge and the working copy instead of requiring a manual review.—

**Comment by bobjacobsen** [comment]
Here's the current Git graph:
skitch
It looks to me like the current master _was_ merged into Pete's master. Yes somehow something went wrong...

**Comment by mattharris** [comment]
@petecressman The problem placeholders were in the `jmri4.5.6.shtml` file, not the `jmri4.5.7.shtml` file that you've changed here.

Out of interest, which tool do you use for interacting with the repository and, when merging, what options does it give when dealing with any conflicts? And which option (if presented) do you choose? This will help use with working out what's maybe going on.

**Comment by petecressman** [comment]
I use Eclipse Egit.  jmri4.5.7.shtml was the only file in staging when i did the commit.  The placeholders were in this file.
 

    On Wednesday, November 30, 2016 2:57 AM, Matthew Harris <notifications@github.com> wrote:
 

 @petecressman The problem placeholders were in the jmri4.5.6.shtml file, not the jmri4.5.7.shtml file that you've changed here.Out of interest, which tool do you use for interacting with the repository and, when merging, what options does it give when dealing with any conflicts? And which option (if presented) do you choose? This will help use with working out what's maybe going on.—

**Comment by mattharris** [comment]
I've handled the changes to `releasenotes/jmri4.5.7.shtml` as part of the release preparation work.

Will bring through the `releasenotes/jmri4.5.6.shtml` updates separately.

I've marked it `WIP` as this will likely not be merged as-is.

**Pull request closed without merging**